gensim/parsing/preprocessing.py

```diff
@@ -63,11 +107,58 @@ def strip_punctuation(s):
 
 
 def strip_tags(s):
+    """Takes string and removes tags.
+
+    Parameters
+    ----------
+    s : str
```

Comment:
Please add description to argument (for example `Input string.`), here and anywhere